Ginger/Unit Tests/NonUITests/ScriptActionTest.cs

```diff
@@ -0,0 +1,77 @@
+﻿#region License
+/*
+Copyright © 2014-2018 European Support Limited
+Licensed under the Apache License, Version 2.0 (the "License")
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at 
+http://www.apache.org/licenses/LICENSE-2.0 
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
+See the License for the specific language governing permissions and 
+limitations under the License. 
+*/
+#endregion
+
+using amdocs.ginger.GingerCoreNET;
+using Amdocs.Ginger.Common;
+using Amdocs.Ginger.CoreNET.Execution;
+using Amdocs.Ginger.CoreNET.Repository;
+using Ginger.Run;
+using GingerCore;
+using GingerCore.Actions;
+using GingerTestHelper;
+using GingerWPF.WorkSpaceLib;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace UnitTests.NonUITests
+{
+    [TestClass]
+    [Level3]
+    public class ScriptActionTest
```

Comment:
We don't need to create new file and class only for script action test.
Please add them in NonDriverAction tests class itself